frmts/stacit/stacitdataset.cpp

```diff
@@ -218,34 +227,43 @@ static void ParseAsset(const CPLJSONObject &jAsset,
         return oProperties[pszName];
     };
 
-    auto oProjEPSG = GetAssetOrFeatureProperty("proj:epsg");
     std::string osProjUserString;
-    if (oProjEPSG.IsValid() && oProjEPSG.GetType() != CPLJSONObject::Type::Null)
+    auto oProjCode = GetAssetOrFeatureProperty("proj:code");
+    if (oProjCode.IsValid() && oProjCode.GetType() != CPLJSONObject::Type::Null)
     {
-        osProjUserString = "EPSG:" + oProjEPSG.ToString();
+        osProjUserString = oProjCode.ToString();
     }
     else
     {
-        auto oProjWKT2 = GetAssetOrFeatureProperty("proj:wkt2");
-        if (oProjWKT2.IsValid() &&
-            oProjWKT2.GetType() == CPLJSONObject::Type::String)
+        auto oProjEPSG = GetAssetOrFeatureProperty("proj:epsg");
```

Comment:
can it be const?
(other `autos` below as well)

Reply:
yes, fixed.